src/containers/main/UpgradeContainer/index.tsx

```diff
@@ -235,6 +235,33 @@ export const UpgradeContainer: FC<IProps> = ({ address, balance }) => {
 		setHideLowBalance(!hideLowBalance);
 	};
 
+	const addToMetamask = async (address: string, symbol: string, token: any) => {
+		if (symbol.length >= 11) {
+			symbol = symbol.slice(4, symbol.length);
+		}
+		//@ts-ignore
```

Comment:
Please remove the @ts-ignore and resolve underlying type error (this should be fixed by correctly typing the token param)   also use ? and ! if the issue is "could be undefined"

Reply:
done